src/main/java/nl/tudelft/semgroup4/collision/DefaultCollisionHandler.java

```diff
@@ -230,4 +248,34 @@ public void run() {
         Game.LOGGER.log(LogSeverity.DEBUG, "Collision", "Player picked up a utility");
         util.activate(game.getCurLevel());
     };
+    
+    private final CollisionHandler<HorMovingWall, AbstractWall> horWallWallHandler = 
+            (game, movingWall, abstractWall) -> {
+        int offset = 5;        
+        
+        if (movingWall.getLocX() < abstractWall.getLocX() 
```

Comment:
I added some comments within the handler to make it a little clearer. I don't think it''s useful to containt it in a different method somewhere, seeing as it it is just a couple of lines, and it's all this handler does.

The offset problem cannot be fixed, and there is no sense in doing it at lower level, because that would make things more complicated. The thing is, the walls move a certain amount of pixels at a time, so when the walls are almost colliding, and then both move 2 pixels in opposite direction (for example) then they overlap for 4 pixels. This is where the offset is needed. 

< logic, organisation